src/Symfony/Component/Console/Tester/Constraint/CommandIsFaulty.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Raphael Stolt <raphael.stolt@gmail.com>
```

Comment:
I don't think this is correct.

Reply:
Which email should go there?

Reply:
The one from Fabien, like in all other Symfony files. You can add yourself as `@author` above the class name if you like